diff --git a/build/AssetUris.js b/build/AssetUris.js
index 1b361a4..f5b5ecf 100644
--- a/build/AssetUris.js
+++ b/build/AssetUris.js
@@ -26,9 +26,6 @@ export function getFileExtension(url) {
  * https://example.com/app/manifest.json, the base URL would be https://example.com/app/. Query
  * parameters and fragments also are removed.
  *
- * For an Expo-hosted project with a manifest hosted at https://exp.host/@user/project/index.exp, the
- * base URL would be https://exp.host/@user/project.
- *
  * We also normalize the "exp" protocol to "http" to handle internal URLs with the Expo schemes used
  * to tell the OS to open the URLs in the the Expo client.
  */
